### DISCUSSION TITLE
Project filter options for search queries in VSCode Pull Request Extension

### DISCUSSION THREAD

**Discussion by cedric-fauth** [community/community#33917] [Projects and Issues]
I tried customizing query settings of the [VSC Github Pull Request Extension](https://github.com/microsoft/vscode-pull-request-github) because I want to only see issues in the current iteration (e.g. current week). The extension is using the Github API but there is no iteration filter I think. 
Is it possible to search and filter issues by project attribute fields? In my project I can filter issues by iteration with something like iteration:@current.
I think this is not possible with the githubIssues.queries setting right now. Are there workarounds using the API?

**Comment by ghost** [comment]
Is there a better way than the following?

1. Schedule a GHA workflow that fetches the [details ](https://docs.github.com/en/issues/planning-and-tracking-with-projects/automating-your-project/using-the-api-to-manage-projects#finding-information-about-items-in-a-project) of all items in a project.
2. Filter this information to identify issues which are in the current iteration
3. [Add a label](https://github.com/andymckay/labeler) `current` to the identified issues.
4. [Filter by label](https://docs.github.com/en/search-github/searching-on-github/searching-issues-and-pull-requests#search-by-label) within the extension.
